modules/c++/math.poly/include/math/poly/Fit.h

```diff
@@ -1,11 +1,17 @@
 #ifndef __MATH_POLY_FIT_H__
 #define __MATH_POLY_FIT_H__
 
+// coda-oss
```

Comment:
Don't actually put comments like this saying where the includes are from.  I do like when they're grouped together with a newline between them like is done here though.